.husky/.gitignore

```diff
@@ -0,0 +1 @@
+_
```

Comment:
Is this file needed? What is this ignoring?

Reply:
Thats a auto-generated file, which ignores the `_`-folder.
In that folder `husky.sh` is located, which gets generated in `prepare`-script.
It seems to be best practice as this all from the husky docs 😅

Reply:
Thanks :)

My husky config is a lot lighter than this so I'm not familiar with most of it.

Reply:
https://www.nhcarrigan.com/code-snippets/#/node-ts-config/husky

This is all my config is lol